src/infosquito/icat.clj

```diff
@@ -349,6 +354,11 @@
        (get-data-objects cfg))
   (log/info "data object indexing complete"))
 
+(defn reindex
```

Comment:
(I just moved this further up in the file, to put it nearer the functions it calls. By and large, stuff above this has to do with the second stage, indexing, while the stuff below it has to do with existence checking during the first stage, purging)